Common/Packets/AlgorithmNodePacket.cs

```diff
@@ -117,6 +117,12 @@ public string AlgorithmId
         [JsonProperty(PropertyName = "sVersion")]
         public string Version;
 
+        /// <summary>
+        /// Virtual environment to use for PythonNet libraries
+        /// </summary>
+        [JsonProperty(PropertyName = "PyVirtualEnv")]
```

Comment:
nit, think `PyVirtualEnv` should follow property name convention in the class, `sPythonVirtualEnvironment` ?